src/components/PersonLink/PersonLink.tsx

```diff
@@ -0,0 +1,63 @@
+import { Link, useParams } from 'react-router-dom';
+import React from 'react';
+import cn from 'classnames';
+import { Person } from '../../types';
+
+type Props = {
+  person: Person;
+  people: Person[];
+};
+
+export const PersonLink: React.FC<Props> = ({ person, people }) => {
+  const { slug } = useParams();
+  const isActivePerson = slug === person.slug;
```

Comment:
Ensure that `slug` is a unique and reliable identifier for each person. If `slug` is not unique, it might cause unexpected behavior when determining the active person.